ansible/tasks/oauth2-proxy.yml

```diff
@@ -0,0 +1,29 @@
+- name: "download oauth2_proxy"
+  get_url:
+    url: "https://github.com/svsticky/oauth2_proxy/releases/download/v2.3/oauth2_proxy"
```

Comment:
There's a somewhat official fork of oauth2_proxy now, with a [new release](https://github.com/pusher/oauth2_proxy/releases/tag/v3.0.0). It's also seeing active development, so we should switch to that one :)